targetconfig.json

```diff
@@ -679,6 +679,12 @@
                     "devUrl": "http://localhost:3000"
                 }
             }
+            "Zukunftswerkstatt-Buchholz/robot-control": {
```

Comment:
The name of the Extension has to be all-lowercase here: "zukunftswerkstatt-buchholz/robot-control"